.devcontainer/setupScripts/runFromAzureTemplate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+#
+# Used on a blank VM Sets via Azure templates to deploy the Mock SpaceStation environment.  Mostly a wrapper for deployGroundStation.sh
+# Syntax: ./runFromAzureTemplate.sh
+
+curl -O "https://raw.githubusercontent.com/azure/mock-spacestation/.devcontainer/setupScripts/deployGroundStation.sh"
+. ./deployGroundStation.sh
```

Comment:
the repo is lightweight enough to clone in its entirety no? this way we don't need to have those encoded strings in a script.

Reply:
or, if we're married to this idea, let's curl the files in `deployFromGroundstation`

Reply:
This is just a wrapper for the Azure Template so we can change/update the deployGroundStation.sh and not affect the template initiation / json file.  More-or-less removes a step in future updates.